lefthook.yml

```
@@ -1,6 +1,10 @@
 pre-commit:
   parallel: true
   commands:
+    test:
+      run: yarn test
+    format:
+      run: yarn format
     lint:
       glob: '*.{ts,tsx}'
       run: yarn eslint {staged_files}
```

```suggestion
      run: yarn eslint --fix {staged_files} && git add {staged_files}
```
Здесь ещё стоит добавить автофикс от линтера и добавление исправленных staged-файлов в коммит.
Сейчас этот хук форматирует файлы в первом коммите, но не добавляет изменения в этот коммит, если таковые были. Приходится делать доп. коммит. Нужно все уместить в один.

Но надо ещё проверить эту команду на винде, с `&&` некоторые команды не работали

---

Еще интересно, как будет работать `parallel`. Если линтер закончит быстрее, чем `format`, то могут быть несоответствия и не включенные в коммит исправленные файлы

---

Попробую сделать один скрипт последовательный на eslint и pritter

---

c windows все ок